cfg/2.0.0/definitions.yaml

```diff
@@ -7988,17 +8122,17 @@ groups:
       tests:
         test_items:
         - flag: ""
-        compare:
-          op: eq
-          value: ""
-        set: true
+          compare:
+            op: eq
+            value: ""
+          set: true
       remediation: |
         Investigate any users with a password change date in the future and correct them. Locking the account, expiring the password, or resetting the password manually may be appropriate.
       scored: true
 
-    - id: 5.4.2
+    - id: 5.4.2.a
       description: "Ensure system accounts are non-login"
-      audit: "egrep -v \"^\\+\" /etc/passwd | awk -F: '($1!=\"root\" && $1!=\"sync\" && $1!=\"shutdown\" && $1!=\"halt\" && $3<500 && $7!=\"/sbin/nologin\" && $7!=\"/bin/false\") {print}'"   
+      audit: "awk -F: '($1!=\"root\" && $1!=\"sync\" && $1!=\"shutdown\" && $1!=\"halt\" && $1!~/^\\+/ && $3<'\"$(awk '/^\\s*UID_MIN/{print $2}' /etc/login.defs)\"' && $7!=\"'\"$(which nologin)\"'\" && $7!=\"/bin/false\") {print}' /etc/passwd"   
       tests:
         test_items:
         - flag: ""
```

Comment:
Similar here, does it work to have an empty flag value like this?

Reply:
Yes